pkg/cmd/server/admin/create_bootstrappolicy_file.go

```diff
@@ -32,15 +33,15 @@ type CreateBootstrapPolicyFileOptions struct {
 	OpenShiftSharedResourcesNamespace string
 }
 
-func NewCommandCreateBootstrapPolicyFile() *cobra.Command {
+func NewCommandCreateBootstrapPolicyFile(commandName string, baseName string, out io.Writer) *cobra.Command {
```

Comment:
And if you were passing information to an API, you wouldn't want to pass FullName, you'd  pass FirstName LastName (or GivenName SurName).  

Should args be: "create-master-certs", "openshift admin create-master-certs" or "create-master-certs", "openshift admin".  I vote for the latter and some commands are taking that, but calling the second arg "fullName" in spite of it being a "baseName".